src/main/resources/templates/categories/televisions/televisions.ftl

```diff
@@ -1,29 +1,18 @@
 <#include '../../layout.ftl'>
```

Comment:
PR already done for this file in [ELG-6](https://github.com/UKGovernmentBEIS/energy-label-service/pull/87)